### PR TITLE
SOC-1565 | Discussions - delete/undelete post

### PR DESCRIPTION
**Pull request by kvas-damian**
https://wikia-inc.atlassian.net/browse/SOC-1565

### CODE REVIEW

**Comment by Bloodmoon24** [comment]
LGTM

File: front/scripts/main/components/post-detail.js [hunk 1]
```diff
@@ -29,6 +36,13 @@ export default App.PostDetailComponent = Ember.Component.extend(
 			return `${Ember.getWithDefault(Mercury, 'wiki.basePath', window.location.origin)}/d/p/${this.get('postId')}`;
 		}),
 
+		checkPermissions(permission) {
```
Comment (rafalkalinski) [inline front/scripts/main/components/post-detail.js]: jsdoc maybe? :)

File: front/scripts/main/mixins/discussion-delete-post.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
+import App from '../app';
+
+export default App.DiscussionDeletePostMixin = Ember.Mixin.create({
+	checkPermissions(post, permission) {
+		return Ember.get(post, '_embedded.userData')[0].permissions.contains(permission);
+	},
+
+	deletePost(post) {
```
Comment (rafalkalinski) [inline front/scripts/main/mixins/discussion-delete-post.js]: jsdoc?

**Comment by rafalkalinski** [comment]
LGTM

**Comment by kvas-damian** [comment]
Will reopen PR to dev

**Pull request closed without merging**